metadata.rb

```diff
@@ -13,8 +14,10 @@
   supports os
 end
 
-depends 'compat_resource', '>= 12.16.3'
 depends 'yum-epel'
+depends 'compat_resource'
```

Comment:
you want the version constraint here so you make sure the user has a working compat_resource